core/src/components/datetime/datetime.tsx

```diff
@@ -171,6 +174,24 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() disabled = false;
 
+  /**
+   * Formatting options, separated by date and time.
+   */
+  @Prop() formatOptions?: DatetimeFormatOptions;
```

Comment:
It would be great to also improve the JSDoc comment here. This is the information that will show in the docs and intellisense. 

Would be good to include what format it requires and maybe a link out to MDN for those rules.

Reply:
i improved the comment